test/tasks/example/sep/007-deputy-pause-key-rotation/README.md

```diff
@@ -0,0 +1,14 @@
+# 005-deputy-pause-key-rotation Deputy Pause Module Rotation key
+Status: [DRAFT]()
+
+## Objective
+
+This is an example task that should be **deleted** once we have real tasks to run. It's purpose is to have a task that runs before the monorepo-integration-test task.
```

Comment:
so do we need this folder?

Reply:
Seems that need to add this into example aswell. 
I don't fully understand why maybe @blmalone knows more here?

Reply:
Every template we add we always need to create an example task that is pinned to a block number. The task gets ran in CI in the `simulate-all-templates` job. Helps make sure templates always work and there are no regressions.

Reply:
Okay I will add the block number of today then we should just need to make sure we have archive node.
Otherwise in the future the CI will fail I think.
Thanks!

Reply:
Yes, just follow whatever pattern we're doing for the other templates.